src/daikinapi.ts

```diff
@@ -29,12 +30,29 @@ export declare const enum TargetHeatingCoolingState {
 
 export type LogMessage = (level: LoggerLevel, message: string, ...parameters: any[]) => void
 
+export type DataChanged = () => void
+
+// After sending an update to the Daikin API it will return old data for up to 15 seconds, so we
+// delay fetching data after an update by this amount. https://daikinone.com/openapi/documentation/
+export const DAIKIN_DEVICE_WRITE_DELAY_MS = 15*1000;
+
+// User is not interacting with a HomeKit controller - background updates for automations
+export const DAIKIN_DEVICE_BACKGROUND_REFRESH_MS = 180*1000;
```

Comment:
This value right now replaces the `refreshInterval` config. We could make it configurable, but 3 minutes is the value Daikin calls out on https://daikinone.com/openapi/documentation/ so hopefully a value that won't ever raise any flags on their end.

Reply:
I'm good removing the refreshInterval.